edq/core/config.py

```diff
@@ -37,6 +39,18 @@ def __eq__(self, other: object) -> bool:
     def __str__(self) -> str:
         return f"({self.label}, {self.path})"
 
+def write_config_to_file(file_path: str, config_to_write: typing.Dict[str, str]) -> None:
```

Comment:
This function name needs some work:
 1) It could be more direct, e.g., "write_config_file".
 2) "Write" without any qualifiers usually implies truncate/overwrite. Maybe you want "update"?